tools/SetupFlow/DevHome.SetupFlow/Views/Summary/SummaryAppsDownloadedReposCloned.xaml

```diff
@@ -0,0 +1,109 @@
+                        NotEmptyValue="Collapsed" />
+                    <converters:BoolToObjectConverter
+                        x:Key="SetUpEnvironmentThirdColumnWidth"
+                        FalseValue="auto"
+                        TrueValue="*" />
+                </ResourceDictionary>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <Grid Margin="0,0,0,30" HorizontalAlignment="Stretch">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="{Binding AppsDownloaded, Converter={StaticResource EmptyCollectionWillNotSpanColumnsConverter}}" />
+            <ColumnDefinition Width="{Binding RepositoriesCloned, Converter={StaticResource EmptyCollectionWillNotSpanColumnsConverter}}" />
+            <ColumnDefinition Width="{Binding WasCreateEnvironmentOperationStarted, Mode=OneWay, Converter={StaticResource SetUpEnvironmentThirdColumnWidth}}" />
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <Grid Grid.Column="0" Visibility="{Binding AppsDownloaded, Converter={StaticResource CollectionVisibilityConverter}}">
+            <Grid.RowDefinitions>
+                <RowDefinition />
+                <RowDefinition />
+            </Grid.RowDefinitions>
+            <TextBlock
+                Grid.Row="0"
+                HorizontalAlignment="center"
+                FontSize="{StaticResource TitleLargeTextBlockFontSize}"
+                FontWeight="{StaticResource InfoBarTitleFontWeight}"
+                Text="{Binding AppsDownloaded.Count}" />
+            <TextBlock
+                Grid.Row="1"
+                HorizontalAlignment="center"
+                Foreground="{ThemeResource TextFillColorSecondary}"
+                Text="{Binding ApplicationsClonedText}" />
+        </Grid>
+        <Grid Grid.Column="1" Visibility="{Binding RepositoriesCloned, Converter={StaticResource CollectionVisibilityConverter}}">
+            <Grid.RowDefinitions>
+                <RowDefinition />
+                <RowDefinition />
+            </Grid.RowDefinitions>
+            <TextBlock
+                Grid.Row="0"
+                HorizontalAlignment="Center"
+                FontSize="{StaticResource TitleLargeTextBlockFontSize}"
+                FontWeight="{StaticResource InfoBarTitleFontWeight}"
+                Text="{Binding RepositoriesCloned.Count}" />
+            <TextBlock
+                Grid.Row="1"
+                HorizontalAlignment="Center"
+                Foreground="{ThemeResource TextFillColorSecondary}"
+                Text="{Binding RepositoriesClonedText}" />
+        </Grid>
+
+        <!--  No apps or repos were cloned.  This UI represents the create an environment flow.  -->
+        <Grid Grid.Column="2" Visibility="{Binding WasCreateEnvironmentOperationStarted, Mode=OneWay}">
```

Comment:
Default mode for Binding is OneWay. Since you sometimes mark that explicitly and sometimes don't, just want to make sure you're expecting that everywhere you use Binding.

Reply:
I'm not going to fix this in this PR.  I do not believe anything will change because the SummaryScreen is static, but I don't want to risk the change.